lib/utils/stringify.ts

```diff
@@ -0,0 +1,30 @@
+import type { Primitive } from 'zod';
```

Comment:
question: just to be sure, we were not meant to import from `zod/core`, right?

Reply:
That is a very good question - checking...

Reply:
Fixed with cbddedb - thank you for catching this issue.

Reply:
Discussed offline - all good.